test/Libraries/Microsoft.Extensions.DataIngestion.Tests/Microsoft.Extensions.DataIngestion.Tests.csproj

```diff
@@ -27,6 +27,10 @@
     <PackageReference Include="OpenTelemetry.Exporter.InMemory" />
   </ItemGroup>
 
+  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net10.0'))">
```

Comment:
This should not be required, as the `Microsoft.Extensions.DataIngestion.csproj` project has this dependency and it's referenced here.